server/plugin/subscriptions.go

```diff
@@ -101,6 +101,8 @@ func (p *Plugin) Subscribe(ctx context.Context, githubClient *github.Client, use
 		return errors.Errorf("invalid repository")
 	}
 
+	owner = strings.ToLower(owner)
```

Comment:
Did you test these changes with a repo that has a lower case letter it it?

Reply:
Hi @cjmartin.  It appears your PR is really close.  Can you address @hanzei's question?